rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -83,7 +83,7 @@
                         </not>
                     </when>
                     <perform>
-                        <hint title="IBM ILog proprietary type reference" effort="0" category-id="potential">
+                        <hint title="IBM ILog proprietary type reference" effort="1" category-id="potential">
```

Comment:
I am leaving 0 at all. Setting them to 1 leaded to an explosion of the story points. Maybe we should generally shift the whole scale so that 1000x 1sp issue which you can handle with a regexp replace wouldn't beat an issue with 64sp which is for several weeks of coding.